concept/type/TypeImpl.java

```diff
@@ -77,6 +79,29 @@ public static TypeImpl of(ConceptProto.Type typeProto) {
         }
     }
 
+    public static ConceptProto.Type.Encoding encoding(Type type) {
```

Comment:
Perhaps we should move this to `Proto.java`.

Reply:
We can't do that, because `Type` is in `//api`, and this method signatures depend on it.